functions-methods.js

```diff
@@ -20,6 +28,24 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail (email) {
+    const zoek_domain = email.indexOf('@');
+    let domain = email.substring(zoek_domain + 1, email.length);
```

Comment:
De tweede parameter is niet nodig in het geval dat je de rest van de string wilt hebben